permission_handler_android/pubspec.yaml

```diff
@@ -1,7 +1,7 @@
 name: permission_handler_android
 description: Permission plugin for Flutter. This plugin provides the Android API to request and check permissions.
 homepage: https://github.com/baseflow/flutter-permission-handler
-version: 10.3.0
```

Comment:
Please include a CHANGELOG entry in `permission_handler_android/CHANGELOG.md`.

Reply:
Done.